### PR TITLE
feat(backend, frontend): add tlms for todos

### PR DESCRIPTION
**Pull request by Haskaris**
Issue ref #5699

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

**Review by gbonaspetti — APPROVED** [review]

File: tools_docker/Debian_Uwsgi_ARM64/Dockerfile [hunk 1]
```diff
@@ -83,7 +83,7 @@ FROM base
 
 LABEL org.opencontainers.image.authors="contact@tracim.fr"
 
-ENV DEFAULT_APP_LIST="contents/thread,contents/file,contents/html-document,contents/folder,contents/kanban,share_content,upload_permission,gallery"
+ENV DEFAULT_APP_LIST="contents/thread,contents/file,contents/html-document,contents/folder,contents/kanban,contents/todo,share_content,upload_permission,gallery"
```
Comment (inkhey) [inline tools_docker/Debian_Uwsgi_ARM64/Dockerfile:86]:
```suggestion
ENV DEFAULT_APP_LIST="contents/thread,contents/file,contents/html-document,contents/folder,contents/kanban,contents/todo,share_content,upload_permission,gallery"
```
Can you verify that `developement.ini.sample` and `config.py` have todo as default app ?

File: backend/tracim_backend/views/core_api/schemas.py [hunk 2]
```diff
@@ -1922,6 +1925,19 @@ class MessageCommentSchema(marshmallow.Schema):
     )
 
 
+class MessageTodoSchema(marshmallow.Schema):
```
[the remark this answers is not in the document]
Comment (Haskaris) [reply backend/tracim_backend/views/core_api/schemas.py:1928]: Done in 5d49f09

**Review by inkhey — APPROVED** [review]

File: backend/tracim_backend/applications/content_todo/controller.py [hunk 3]
```diff
@@ -172,10 +178,11 @@ def delete_todo(self, context, request: TracimRequest, hapic_data=None):
             current_user=request.current_user, session=request.dbsession, config=app_config,
         )
 
-        todo = None  # type: typing.Optional['Todo']
+        todo_content = None  # type: typing.Optional['Content']
```
Comment (grignards) [inline backend/tracim_backend/applications/content_todo/controller.py:181]: I think this line is useless.

**Review by grignards — APPROVED** [review]

File: backend/tracim_backend/applications/content_todo/controller.py (hunk 3 above)
Comment (Haskaris) [reply backend/tracim_backend/applications/content_todo/controller.py:181]: If I don't put this declaration, the `todo_content` variable is defined inside the `try` below (line 182)
However I need this variable out of the `try` int the `with` (line 192)

I'm not sure I can remove this line safely

Comment (grignards) [reply backend/tracim_backend/applications/content_todo/controller.py:181]: It is safe as if the content is not found, an exception is raised (line 187) so the code afterward is not executed.

**Pull request merged**